src/cpp/handler/handlermain.cpp

```diff
@@ -44,6 +41,12 @@ public slots:
 		delete app;
 		QCoreApplication::exit(returnCode);
 	}
+	
+	~HandlerAppMain()
+	{
+		app->~HandlerApp();
```

Comment:
C++ destructors shouldn't be called directly. The `delete app` below will do it.